lcls-twincat-motion/Library/POUs/Motion/States/FB_EpicsInOut.TcPOU

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_EpicsInOut" Id="{13bd60fa-f200-43aa-9bf2-b14d410be016}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_EpicsInOut
+// Example usage of FB_PositionStateManager for a simple IN/OUT axis. See NOTE: comments.
```

Comment:
Looking forward to seeing the pmps version of this.

Reply:
This is the PMPS version, you just need to set whether or not the destination states are safe to move to in your PMPS governer